CLI.md

```diff
@@ -290,7 +294,20 @@ Flags:
       --supply int                 Max Token supply, use -1 for unlimited
       --symbol string              Optional abbreviated token symbol
       --type <"FAT-0" | "FAT-1">   Token standard to use
+```
+
+**Example Commands**
+
+Initialize a FAT-0 token called "test" with a maximum supply of 100,000 units:
+
+```
+fat-cli issue --ecadr EC3cQ1QnsE5rKWR1B5mzVHdTkAReK5kJwaQn5meXzU9wANyk7Aej --sk1 sk1... --identity 888888a37cbf303c0bfc8d0cc7e77885c42000b757bd4d9e659de994477a0904 --tokenid test --type "FAT-0" --supply 100000
+```
 
+Initialize a FAT-1 token called "test-nft" with an unlimited supply.
```

Comment:
On the above you used a colon at the end of your sentence and here a period. Maybe chose one or the other to be consistent :)